lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -287,7 +287,8 @@
     - Linux based OSes requires Perl package to be installed for OS customizations.
     - 'Common parameters (Linux/Windows):'
     - ' - C(dns_servers) (list): List of DNS servers to configure.'
-    - ' - C(dns_suffix) (list): List of domain suffixes, also known as DNS search path (default: C(domain) parameter).'
+    - ' - C(dns_suffix) (list): List of domain suffixes, also known as DNS search path (default: C(domain) parameter).
+          Also accepts space separated domain suffixes.'
```

Comment:
I would not advertise space-delimited lists as this is only for backward compatibility. We don't want to offer people this choice, only support people that did not have the choice before. I would even deprecate space-delimited lists over time.